src/Factory.php

```diff
@@ -32,7 +32,7 @@ class Factory extends BaseFactory
      */
     public function writer($path = TMP)
     {
-        return parent::writer($path);
+        return parent::writer($this->config->get('general.timestampPath') ?: $path);
```

Comment:
What temporary files? I'm not seeing any in my setup, but perhaps I'm missing a configuration setting somewhere. I'll run some tests.